SimTracker/TrackTriggerAssociation/plugins/CleanAssoc.cc

```diff
@@ -0,0 +1,83 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
```

Comment:
If the purpose of this class is to produce a reduced version of the Cluster association map, then it needs renaming. The file name should mention "Clus".

Reply:
will be obsolete soon